bin/level/app.rs

```diff
@@ -17,6 +20,10 @@ pub struct LevelView<R: gfx::Resources> {
     _level: level::Level,
     cam: space::Camera,
     input: Input,
+
+    _last_mouse_pos: cgmath::Vector2<f32>,
```

Comment:
shouldn't use `_` prefix, since these are actually used (in Rust, it's only needed to silence compiler warnings on unused stuff)

Reply:
thought that private variables naming convention

Reply:
FYI, private is everything that is not qualified with `pub`